inc/define.php

```diff
@@ -104,6 +104,7 @@
             'hr_HR' => ['Hrvatski',             'hr_HR.mo','hr',    'hr','croatian'   , 2],
             'hu_HU' => ['Magyar',               'hu_HU.mo','hu',    'hu','hungarian'  , 2],
             'it_IT' => ['Italiano',             'it_IT.mo','it',    'it','italian'    , 2],
+            'ka'    => ['ಕನ್ನಡ',                 'ka.mo',   'en-GB', 'en', 'kannada'   , 2],
```

Comment:
`Kannada` the code is `kn`, `ka` is the code for `Georgian` language (see)[https://www.transifex.com/explore/languages/].
As all mo files are using the lang_VARIANT format, maybe we should name it `kn_IN.mo`.

Reply:
Well catched... As for the variant, yes, but the one added on transifex was "kn" :/ (I do not remember It's been requested, it may be old)